obal/data/roles/build_package/meta/main.yml

```diff
@@ -1,6 +1,4 @@
 ---
 dependencies:
   - { role: setup_workspace }
-  - { role: ensure_package }
   - { role: setup_sources }
-  - { role: package_variables }
```

Comment:
these two are not required anymore after your cleanup in #182, right?

Reply:
They are still depended upon, but it's not necessary to specify them.

Role dependency resolves as follows:

```
build_package
|-- setup_workspace
 `- setup_sources
    |-- git_annex_setup
    |    `- package_variables
     `- spec_file
         `- ensure_package
             `- package_variables
```

We don't need to specify `package_variables` or `ensure_package` since we've already got `setup_sources`.

Reply:
I would also have believed you, if you just wrote "yes" ;)